### DISCUSSION TITLE
Official Mono.Debugger.Soft.dll download link

### DISCUSSION THREAD

**Discussion by robertthiemann** [mono/mono#20794] [General]
Hi,

I wanted to have a look at the Mono.Debugger.Soft.dll but i can't find any official place to download it.
I'm probably looking in the wrong places though...

Please help!

Cheers,
Robert.

**Comment by migueldeicaza** [comment]
It is part of any Mono binary installation, we do not distribute that as a Nuget or anything like that.

It can also be built from the repository, if you have built Mono, the code is in mono/mcs/class/Mono.Debugger.Soft
